todo_app/data/session_items.py

```diff
@@ -171,3 +204,56 @@ def markcomplete(itemid):
       headers=headers,
       params=query
    )
+
+
+#######Create a new board ######
+def create_trello_board():
+   url = "https://api.trello.com/1/boards/"
+
+
+
+   query = {
+      'key': os.getenv('KEY'),
+      'token': os.getenv('TOKEN'),
+      'name': 'Assignments'
+   }
+
+
+   response = requests.request(
+      "POST",
+      url,
+      params=query
+   )
+   getresponse = response.text
+   
+   getresponse1 = json.loads(getresponse)
+
+   idofboardcreated = getresponse1.get('id')
+   
+   return idofboardcreated
+   
+
+
+
+#######Delete the board ######
+
+def delete_trello_board():
+   load_dotenv()
+   idtobedeleted = os.getenv('TRELLO_BOARD_ID')
```

Comment:
We shouldn't need to load_dotenv here, since the system must already have loaded in the key/token beforehand to create a board. In addition, it's a little odd that this code knows about an environment variable (`TRELLO_BOARD_ID`) that is only set in a test - instead it might be nicer to have the board id be passed through as a function argument (since the test already knows what is)

Reply:
Removed from here